src/gobupload/storage/handler.py

```diff
@@ -150,13 +150,7 @@ def _init_views(self):
         for catalog in views.get_catalogs():
             for entity in views.get_entities(catalog):
                 for view_name, view in views.get_views(catalog, entity).items():
-                    if view['version'] != "0.1":
-                        # No migrations defined yet...
-                        raise ValueError(
-                            "Unexpected version, please write a generic migration here or migrate the view"
-                        )
-
-                    self._create_view(f"{catalog}_{entity}_{view_name}", "\n".join(view['query']))
+                    self._create_view(view['name'], view['query'])
```

Comment:
Why delete the version check?

Reply:
Because it's not trivial, and I don't think it's needed now. Views are updated regularly, yet they all still have version 0.1. We don't have any process in place for updating views to a higher version (what would we do, and when do we update?). The views versions currently don't have a meaning, and the hardcoded check  on it being 0.1 (as was in place) doesn't solve or enforce anything.

Adding the version number to the .sql files seemed to introduce other challenges, and hardcoding 0.1 in the sql reader file would be useless (we'll be comparing one hardcoded value to another..). Therefore I thought I'd leave it out until the need for versioning views arises, which may never be the case. Imo no need to engineer a complex suboptimal solution now for something that may or may not be needed in the future.